### PR TITLE
Capture JS sandbox logs and display them in the UI for the conversation.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

File: go/apps/jsbox/tests/test_log.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+from twisted.trial.unittest import TestCase
+from twisted.internet.defer import inlineCallbacks, returnValue
+
+from vumi.application.tests.test_sandbox import (
+    ResourceTestCaseBase, DummyAppWorker)
+
+from go.apps.jsbox.log import LogManager, GoLoggingResource
+
+
+class TestLogManager(TestCase):
```
Comment (smn) [inline go/apps/jsbox/tests/test_log.py]: Where is this being used?

**Comment by hodgestar** [comment]
For now I'm limiting this to the non-UI part. Will create a separate ticket and PR for that.

**Comment by smn** [comment]
:+1: in general though I'm a bit concerned about the `RedisManager` prefix being hardcoded in the view but the config must be made matching for the `LoggingResource` & the redis manager that's being handed to the `LogManager`.

**Comment by hodgestar** [comment]
Updated -- could I get a re-review?

**Comment by smn** [comment]
:+1:

**Pull request merged**